### PR TITLE
add ycsb4tera

### PR DESCRIPTION
**Pull request by 00k**
use build_ycsb.sh to build YCSB
use load_data.sh to init table (put)
use run_test.sh to run test (put/get)

### CODE REVIEW

File: benchmark/ycsb4tera/tera/src/main/java/com/yahoo/ycsb/TeraDB.java [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+
+/**
+ * Basic DB that just prints out the requested operations, instead of doing them against a database.
+ */
+public class TeraDB extends DB
+{
+        public static final String VERBOSE="basicdb.verbose";
+        public static final String VERBOSE_DEFAULT="false";
+        
+        public static final String SIMULATE_DELAY="basicdb.simulatedelay";
```
Comment (caijieming-baidu) [inline benchmark/ycsb4tera/tera/src/main/java/com/yahoo/ycsb/TeraDB.java]: java风格的tab都是8字节吗

**Comment by caijieming-baidu** [comment]
LGTM

**Comment by caijieming-baidu** [comment]
ref: http://maven.apache.org/guides/index.html

**Comment by taocp** [comment]
LGTM

File: benchmark/ycsb4tera/tera/src/main/java/com/yahoo/ycsb/TeraDB.java (hunk 1 above)
Comment (00k) [inline benchmark/ycsb4tera/tera/src/main/java/com/yahoo/ycsb/TeraDB.java]: 不知道，ycsb是8

**Pull request merged**